Python/other/paranthesis_checker.py

```diff
@@ -0,0 +1,33 @@
+def checkParanthesis(input_string):
+    ''' 
+        argument : string
+
+        Function checks whether the Brackets Combinations Provided As Input 
+        is Balanced or Not.
+        
+        usage : checkParanthesis(string)
+        
+    '''
+
```

Comment:
Remove these blank lines

Reply:
@HarshCasper I have completed your desired changes .Kindly look through it.